src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.WorkspaceHostState.cs

```diff
@@ -139,7 +139,7 @@ internal void SolutionClosed()
 
                     var solutionInfo = SolutionInfo.Create(id, version.Value, solutionFilePath, projects: projectInfos);
 
-                    _tracker.NotifyWorkspaceHosts(host => host.OnSolutionAdded(solutionInfo));
+                    _workspaceHost.OnSolutionAdded(solutionInfo);
```

Comment:
We are already in WorkspaceHostState, we don't want to notify all the *other* hosts that we're adding this SOlutionInfo to this host.